client/payment-request/utils/utils.js

```diff
@@ -0,0 +1,117 @@
+/* global wcpayPaymentRequestParams */
+
+/**
+ * Retrieves payment request data from global variable.
+ *
+ * @param {string} key The object property key.
+ * @return {mixed} Value of the object prop or null.
+ */
+export const getPaymentRequestData = ( key ) => {
+	if (
+		'object' === typeof wcpayPaymentRequestParams &&
+		wcpayPaymentRequestParams.hasOwnProperty( key )
+	) {
+		return wcpayPaymentRequestParams[ key ];
+	}
+	return null;
+};
```

Comment:
I thought it would be "uglier" to add the wcpayPaymentRequestParams global variable in React everywhere, so I added this "getter function", so that we know where it's coming from. It's supposed to look similar to `getStripeServerData` from the Stripe implementation.

Do you think it's better to use `wcpayPaymentRequestParams?.key` along with `/* global wcpayPaymentRequestParams */` at the header instead? I'm happy to change.

Reply:
Ahhh, that makes sense actually; wrapping the global variable like that. I didn't connect the dots there.

I think I'd prefer to keep the function, provided that we make it clearer in the function body that `wcpayPaymentRequestParams` is a global? Possibly with a `global wcpayPaymentRequestParams` at the top of the function — if that doesn't break anything — or a comment?

Reply:
There's a `/* global wcpayPaymentRequestParams */` comment at the top, where we usually place this sort of comment. Marking as resolved.